packages/usa-layout-docs/src/styles/_usa-layout-docs.scss

```diff
@@ -1,17 +1,9 @@
 @use "uswds-core" as *;
 
-// Flexbox positioning to move sidenav below main content on small screens
 .usa-layout-docs__sidenav {
-  order: 2;
-  padding-top: units(4);
+  padding-bottom: units(4);
 
   @include at-media("desktop") {
-    padding-top: 0;
-  }
-}
-
-.usa-layout-docs__main {
```

Comment:
This rule is no longer in the CSS but it still exists in the markup in [usa-docs-inner.twig](https://github.com/uswds/uswds/blob/develop/packages/templates/usa-documentation/includes/_usa-docs-inner.twig#L52). The markup there also adds an undefined `usa-layout-docs` class as well. Could you remove those as well just to keep things tidy? :broom:

Reply:
Removed rule. 

Also, `usa-layout-docs` in 91213f6 because its placement/name seems off based on BEM standards.